### DISCUSSION TITLE
Does multi-aspect modeling support hierarchy for each representation?

### DISCUSSION THREAD

**Discussion by staedi** [MaartenGr/BERTopic#1308] [Q&A] [answered]
Hi Maarten,

First, Thanks for the recent update of v0.15 (for *exponential backoff* mainly).

I'm trying to trying out multi-aspect topic modeling for my use case.

I use multiple representation models for the same text data so multi-aspect seems beneficial.
But, the main focus for my case is hierarchical topic representations (`hierarchical_topics()`).

It seems that as `hierarchical_topics()` only work for the BERTopic object, I couldn't find a way to apply it to each representation model of a multi-aspect BERTopic object.

Thanks in advance.

**Comment by MaartenGr** [comment, marked as answer]
Thanks for the kind words!

Indeed, multi-aspect topic modeling is currently only implemented in v0.15 for the main `fit`/`fit_transform` functions and some visualizations. There are quite a few specifics to work through to enable it for hierarchical, dynamic, class-based, topic distributions, and a few other functions. Since that is quite the task by itself, I left it for the release after the current.

**Reply by staedi** [reply]
Thanks Maarten for the quick response!

Then, I'll keep using the same method for the time being.
